lightningd/datastore.c

```diff
@@ -298,34 +298,40 @@ static struct command_result *json_datastoreusage(struct command *cmd,
 	const char **k, **key = NULL;
 	struct db_stmt *stmt;
 	const u8 *data;
-	u64 gen;
-	u64 total = 0;
+	u64 gen, total_bytes = 0;
 
```

Comment:
Combine first two commits, since the first doesn't compile anywayk, and you radically change behavior.

Also **key doesn't need initialization.